src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -281,6 +281,8 @@ variables {f g}
 
 /-! ### The identity normed group hom -/
 
+variable (V)
```

Comment:
It seems more reasonable to me to have `V` explicit in the definition of `normed_group_hom.id`, but I am not 100% sure.